public/styles/styles.css

```diff
@@ -0,0 +1,10 @@
+.book-cover{
+  height: 275px;
+  width: 200px;
+}
+.editing{
+  display: inline;
+}
+.title{
+  width: 550px;
+}
```

Comment:
look up https://bestmotherfucking.website/ for styling tips, but great job styling